### PR TITLE
Feature invoiceninja

### PR DESCRIPTION
**Pull request by maximuskowalski**
# Description

Invoice Ninja Saltbox convert

# How Has This Been Tested?

- [x] Install on saltbox server and configure with user
- [x] test logging in

# New Role Checklist:

- [x] I have updated the header in any files I may have used as templates with my own information
- [x] I have added my new role to `community.yml`
- [ ] I have verified that any Docker images used are current and supported.

### CODE REVIEW

**Comment by saltydk** [comment]
https://github.com/anojht/invoiceninja maybe?

**Comment by RXWatcher** [comment]
I'm an idiot. I was looking at his compose example and it was the same as
the official and needed 2 containers. You're correct and if you look at his
readme, it's one.

File: roles/invoiceninja/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  - "{{ invoiceninja_paths_location }}/logs"
+  - "{{ invoiceninja_paths_location }}/logs/cron"
+  - "{{ invoiceninja_paths_location }}/logs/nginx"
+invoiceninja_paths_config_location: "{{ invoiceninja_paths_location }}/env"
+
+################################
+# Web
+################################
+
+invoiceninja_web_subdomain: "{{ invoiceninja.subdomain|default('invoiceninja',true) }}"
```
Comment (saltydk) [inline roles/invoiceninja/defaults/main.yml:35]: We're going away from this type of stuff. Overwrites will only happen in inventory.

**Comment by saltydk** [comment]
Status?

**Comment by maximuskowalski** [comment]
I will not be doing any more work on this. If nobody wants to pick it up we can just close it.

**Pull request closed without merging**